core/cfdefs/cfdef_standard.php

```diff
@@ -479,6 +481,9 @@ function cfdef_input_textbox( array $p_field_def, $p_custom_field_value, $p_requ
  */
 function cfdef_input_textarea( array $p_field_def, $p_custom_field_value, $p_required = '' ) {
 	echo '<textarea class="form-control" ' . helper_get_tab_index() . ' id="custom_field_' . $p_field_def['id'] . '" name="custom_field_' . $p_field_def['id'] . '"' . $p_required;
+	if( 0 < $p_field_def['length_max'] ) {
```

Comment:
It is much more readable to have length_max > 0.

Reply:
ok